### DISCUSSION TITLE
Can torchaudio be added to https://repo.radeon.com/rocm/manylinux/rocm-rel-*?

### DISCUSSION THREAD

**Discussion by npuichigo** [ROCm/ROCm#3337] [Q&A] [answered]
Until now, only torchvision is built and included in repo.radeon.com, but torchaudio also needs compilation and is provided in https://download.pytorch.org/whl/rocm6.0. Why not add torchaudio?

**Comment by darren-amd** [comment, marked as answer]
Hi @npuichigo,

Thanks for the suggestion, I spoke with the internal team and we will be adding the wheels package to the Radeon repo starting from ROCm 6.3.
